src/languages/es.js

```diff
@@ -371,6 +371,16 @@ export default {
             certify: 'Debe certificar que la información es verdadera y precisa',
         },
     },
+
+    // TODO: Waiting translations
```

Comment:
Could we create an issue for this, rather than leaving a TODO here that will probably get ignored?

Reply:
Oh I see, you'll hold the PR on this instead?